src/test/javascript/spec/component/exercises/quiz/quiz-participation.component.spec.ts

```diff
@@ -294,7 +292,7 @@ describe('QuizParticipationComponent', () => {
 
         it('should react to errors', () => {
             const alertService = fixture.debugElement.injector.get(AlertService);
-            const alertSpy = jest.spyOn(alertService, 'error').mockReturnValue({ msg: '' } as any);
+            const alertSpy = jest.spyOn(alertService, 'addAlert').mockReturnValue({ msg: '' } as any);
```

Comment:
This is not a proper return value for the method

Reply:
True. We don't even need any returned value here, I just removed it